contrib/localnet/scripts/start-zetacored.sh

```diff
@@ -246,6 +246,9 @@ then
 # solana tester
   address=$(yq -r '.additional_accounts.user_solana.bech32_address' /root/config.yml)
   zetacored add-genesis-account "$address" 100000000000000000000000000azeta
+# ton tester
+  address=$(yq -r '.additional_accounts.user_ton.bech32_address' /root/config.yml)
+  zetacored add-genesis-account "$address" 100000000000000000000000000azeta
```

Comment:
```suggestion
  zetacored add-genesis-account "$address" 10000000000000000000000000azeta
```
Something I fixed in https://github.com/zeta-chain/node/pull/3595/commits/18009d279c44ddf64333d80b10855815094838f7 we were giving too many ZETA to genesis accounts that we were reaching the hard ZETA supply cap, preventing some tests to run